src/app/pages/vm/devices/device-add/device-add.component.ts

```diff
@@ -402,11 +404,16 @@ export class DeviceAddComponent implements OnInit, OnDestroy {
 
     // pci
     this.ws.call('vm.device.passthrough_device_choices').pipe(untilDestroyed(this)).subscribe((res) => {
+      this.isLoadingPci = true;
       this.pptdev = _.find(this.pciFieldConfig, { name: 'pptdev' }) as FormSelectConfig;
       this.pptdev.options = Object.keys(res || {}).map((pptdevId) => ({
         label: pptdevId,
         value: pptdevId,
       }));
+      this.isLoadingPci = false;
+    },
+    () => {
```

Comment:
We should handle the error case here as well since we have added a specific call back, using `EntityUtils`.